### DISCUSSION TITLE
How to get annotation files for video?

### DISCUSSION THREAD

**Discussion by Hito-HA** [bdd100k/bdd100k#95] [Q&A] [answered]
Hi,I'm strongly interested in this dataset.
I can see labes files for images(bdd100k_labels_images_train.json and bdd100k_labels_images_val.json),
and I'm wondering where can I get labels files for videos?

**Comment by fyu** [comment, marked as answer]
What labels are you referring to? This page provides a menu of available downloads and labels: https://doc.bdd100k.com/download.html.

**Reply by Hito-HA** [reply]
Thank you for your reply.
On the above portal(https://doc.bdd100k.com/download.html), there is "videos" folder and in this folder, there are 70K movies for train and 10K movies for val.  So, I'm looking for labels for these videos.
Regards.

**Reply by fyu** [reply]
The same page also lists different types of labels. You can look them up.

**Reply by Hito-HA** [reply]
I could find video data in "MOT2020Images" folder and corresponding box annotations in "MOT2020labes" folder.
Thank you so much.
